bothub/settings.py

```diff
@@ -394,12 +397,33 @@
 
 BASE_MIGRATIONS_TYPES = ["bothub.common.migrate_classifiers.wit.WitType"]
 
+
 # Suggestion Languages
 SUGGESTION_LANGUAGES = env.str("SUGGESTION_LANGUAGES")
 
+
 # Word suggestions
-N_WORDS_TO_GENERATE = env.str("N_WORDS_TO_GENERATE")
+N_WORDS_TO_GENERATE = env.int("N_WORDS_TO_GENERATE")
+
+
+# Intent suggestions
+N_SENTENCES_TO_GENERATE = env.int("N_SENTENCES_TO_GENERATE")
+
 
+# django_redis
+CACHES = {
+    "default": {
+        "BACKEND": "django_redis.cache.RedisCache",
+        "LOCATION": "redis://127.0.0.1:6379/1",
```

Comment:
for a production environment we need environment variables, add an environment variable for the redis